test/Libraries/RevitTestServices/RevitSystemTestBase.cs

```diff
@@ -276,7 +276,8 @@ protected override void StartDynamo(TestSessionConfiguration testConfig)
                         SchedulerThread = new TestSchedulerThread(),
                         PackageManagerAddress = "https://www.dynamopackages.com",
                         ExternalCommandData = commandData,
-                        ProcessMode = RevitTaskProcessMode
+                        ProcessMode = RevitTaskProcessMode,
+                        Preferences = PreferenceSettings.Instance
```

Comment:
have you tried running all RTF tests locally?

Reply:
> have you tried running all RTF tests locally?

I have tried but even when I put the paths and the proper tests it got me errors, I have tested with some Dynamo flows under the DynamoRevit pointing to DynamoCore 4.8